src/app/markdown/markdown.component.html

```diff
@@ -53,7 +54,7 @@
   <button
     (click)="saveClick()"
     class="fl btn btn-primary pull-right action-btn btn-save"
-    [disabled]="saving"
+    [disabled]="saving || (!allowEmptySave && fieldEmpty)"
```

Comment:
Same conditions should check for the disabled class as well in line no. 58.

Reply:
We don't actually need a class. It's a button so `disabled` property should work by itself